code/modules/jobs/job_types/scientist.dm

```diff
@@ -4,7 +4,7 @@
 	department_head = list("Research Director")
 	department_flag = MEDSCI
 	faction = "Station"
-	total_positions = 5
+	total_positions = 8
```

Comment:
even 5 scientists roundstart is a annoying number because there are finite things you can do, it will be absolutely abysmal at 8